web/sites/default/config/webform.webform.report_campsite.yml

```diff
@@ -645,15 +652,15 @@ handlers:
       requester_name: contact_name
       requester_email: contact_email
       subject: 'Abandoned Vehicle Report'
-      comment: "<h2>Location</h2>\r\n[webform_submission:values:report_location:html]\r\n<h2>Vehicle info</h2>\r\n<strong>Color:</strong> [webform_submission:values:report_vehicle:color]\r\n<br>\r\n<strong>License Plate Number:</strong> [webform_submission:values:report_vehicle:license_plate_number]\r\n<br>\r\n<strong>License Plate State:</strong> [webform_submission:values:report_vehicle:license_plate_state]\r\n<br>\r\n<strong>Make:</strong> [webform_submission:values:report_vehicle:make]\r\n<br>\r\n<strong>Type:</strong> [webform_submission:values:report_vehicle:type]"
-      tags: 'drupal webform'
+      comment: "<h2>Location</h2>\r\n[webform_submission:values:report_location:html]\r\n<h2>Vehicle info</h2>\r\n<strong>Color:</strong> [webform_submission:values:report_vehicle:color]\r\n<br>\r\n<strong>License Plate Number:</strong> [webform_submission:values:report_vehicle:license_plate_number]\r\n<br>\r\n<strong>License Plate State:</strong> [webform_submission:values:report_vehicle:license_plate_state]\r\n<br>\r\n<strong>Make:</strong> [webform_submission:values:report_vehicle:make]\r\n<br>\r\n<strong>Type:</strong> [webform_submission:values:report_vehicle:type]\r\n<br>\r\n<strong>Related campsite report:</strong> [webform_submission:values:report_ticket_id]"
+      tags: 'drupal webform campsite'
```

Comment:
Adds the "campsite" tag to Abandoned Autos tickets (this handler is currently disabled).